CredentialPostTest/CredentialPostTest/wwwroot/js/site.js

```diff
@@ -2,3 +2,26 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your Javascript code.
+function bindEvent(element, eventName, eventHandler) {
+    if (element.addEventListener) {
+        element.addEventListener(eventName, eventHandler, false);
+    } else if (element.attachEvent) {
+        element.attachEvent('on' + eventName, eventHandler);
+    }
+}
+
+//here goes the code which should be run on iframe parent when integration is created 
+function processResult(result) {
+    let systems = result.split('|');
```

Comment:
We subscribed to all `message` events here, so we can receive not only messages from iFrame. At least we need to add check that `systems` contains 2 elements and process this case further.

Reply:
fixed